rockusb/src/protocol.rs

```diff
@@ -173,6 +183,7 @@ pub struct CommandBlock {
     cdb_length: u8,
     // Command data block fields
     cd_code: CommandCode,
+    cd_reserved: u8,
```

Comment:
if it's used it's clearly not reserved

Reply:
it's an optional parameter to the command; in rkdeveloptool they call it a "subcode". What do you think about `cd_subcode` ?